src/style.css

```diff
@@ -0,0 +1,19 @@
+body {
+  font-family: Arial, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
```

Comment:
do not style tag, use class for it.
exclusion `body`

Reply:
I have same problem in previous task, and don't understand exactly how I should solve it